### DISCUSSION TITLE
Zendesk crawler integration

### DISCUSSION THREAD

**Discussion by oliver-gray** [GitbookIO/community#553] [Feature Requests]
Zendesk is a widely used platform for providing information to product costumers/consumers.

The Zendesk crawler is able to read to content of webpages and redirect costumers with queries to the relevant webpages. 

In order for the crawler to be able to read webpage content, webpage ownership must be established. This is done by the insertion of the following html tag <meta name='zd-site-verification' content='mjxv8p9fvi9yv7k3v9jtsb' /> onto the page.

John from GitBook informed me that inserting the tag in the header section of GitBook markdown would not edit the html tag on the site, and that instead the GitBook team would have to generate the integration.

The Zendesk crawler does not require any further inputs into the GitBook code. 

Here is some relevant documentation on the search crawler:
https://support.zendesk.com/hc/en-us/articles/4593564000410-Setting-up-the-search-crawler
https://support.zendesk.com/hc/en-us/articles/4593607724186-Troubleshooting-the-search-crawler

**Comment by aikoii** [comment]
Hey 👋! Seems like Zendesk no longer requires a verification script to crawl external sites. You can add your published URL or your sitemap `[your-docs-url]/sitemap.xml` and it will handle indexing from there

**Comment by oliver-gray** [comment]
Thanks for the heads up. 
We'll take a look.

**Comment by oliver-gray** [comment]
Thanks! This has been resolved. 
Thanks for keeping an eye on it.

**Reply by aikoii** [reply]
This is great @oliver-gray! Quick question, are you on Enterprise Zendesk, or does this work across all plans? 🙏

**Reply by oliver-gray** [reply]
We are on Enterprise Zendesk. I'm not sure about other plans.

**Reply by aikoii** [reply]
Good to know - thank you! 👍
